src/Uno.UWP/Storage/Streams/FileInputStream.cs

```diff
@@ -0,0 +1,29 @@
+#nullable enable
+
+using System;
+using System.IO;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public sealed partial class FileInputStream : IInputStream, IDisposable, IStreamWrapper
+	{
+		private readonly Stream _stream;
+
+		internal FileInputStream(string path, ulong position)
+		{
+			_stream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
Are you sure for the `FIleShare.Read` option?

Reply:
Nope ... fixed!